AGGREGATE.md

```diff
@@ -73,31 +73,25 @@ on the response to the `attributionsrc` request:
 list.
 ```jsonc
 [{
-  // Generates a "101011001" key prefix named "campaignCounts"
+  // Generates a "0x159" key prefix named "campaignCounts"
   "id": "campaignCounts",
-  "key_piece": "101011001", // User saw ad from campaign 345 (out of 511)
-  "key_offset": 0
+  "key_piece": "0x159", // User saw ad from campaign 345 (out of 511)
```

Comment:
Shall we say (out of 4095 since it's 4*3=12 bits)?

Reply:
No, because 511 campaigns is why we need a 9 bit offset below. I clarified all this in the comments